README.md

```diff
@@ -32,6 +32,14 @@ bindsym $mod+p exec i3_quickterm
 # always pop standard shell, without the menu
 bindsym $mod+b exec i3_quickterm shell
 ```
+Building From Source
```

Comment:
There is not much to build, as this is a python script. This section should probably just be called "Installation".